data/language/dutch.txt

```diff
@@ -584,7 +584,7 @@ STR_0582    :
 STR_0583    :
 STR_0584    :
 STR_0585    :
-STR_0586    :Bootvormige karretjes lopen over achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
+STR_0586    :Bootvormige karretjes rijden over een achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
```

Comment:
Het 2de punt van Gymnasiast klopt. In het geval van rail als spoorstaaf ben ik het met je eens, maar in het geval van 'achtbaanrails' is het anders. Achtbaanrail bestaat helemaal niet (dat heet dan monorail zoals je zelf al aanstipt) en achtbaanrails is gewoon enkelvoud en staat voor de achtbaanrails van normale achtbanen zoals deze achtbaan: http://www.rides.nl/images/items/264/21749.800.jpg
In het Engels is het ook gewoon enkelvoud, en ik heb er 'een' voorgezet om duidelijk te maken dat het om enkelvoud gaat, wat ook wel nodig blijkt. Al kan het natuurlijk wel gewoon zonder lidwoord.

Ik heb inderdaad ook bij Van Dale gekeken, maar daar zeggen ze natuurlijk niks over 'achtbaanrails'. De online versie is sowieso heel beknopt, en over 'rails' zijn we het nu wel eens.